docs/testrunner-toolkit/saucectl.md

```diff
@@ -1,188 +1,355 @@
+## `$ saucectl init [flags]`
 
-__Description__: Sets your [Sauce Labs authentication credentials](https://app.saucelabs.com/user-settings) and generates a `credentials.yml` file. Please refer to [the installation page](/testrunner-toolkit/installation#connecting-to-sauce-labs) for further information.
+A bootstrapping command to generate a configuration file for the framework of your choice in the format required by `saucectl`. Running the command with no parameters initiates interactive mode, prompting you for relevant property values iteratively. Alternatively, you can add supported parameters inline to specify the relevant property values.
 
-__Example__:
-```bash
-saucectl configure
+In either case, the command generates a `.sauce/config.yml` folder and file in the location from which the command is run. If you have an existing project directory for your framework, it is advised that you run this command from the project root.
+
+```bash title="Interactive Example"
+$ saucectl init
+12:13:20 INF Start Init Command
+? Select region: us-west-1
+? Select framework:  [Use arrows to move, type to filter]
+> cypress
+ playwright
+ puppeteer
+ testcafe
+ espresso
+ xcuitest
+```
+
+```bash title="Batch Example"
+$saucectl init -r us-west-1 -f cypress -b chrome
 ```
 
-### `run`
+`saucectl` supports the following configuration flags as inline specifications.
+
+
+### `-a, --accessKey`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The authentication access key associated with the Sauce Labs user account making this request. If you have not set your authentication credentials as environment parameters or generated a `credentials.yml` file, this value is required.
+
+---
+
+### `-app`
+<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
+
+The path to a valid mobile application to test.
+
+---
+
+### `--artifacts.download.when`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies when and under what circumstances to download artifacts. Valid values are:
+
+* `always`: Always download artifacts.
+* `never`: Never download artifacts.
+* `pass`: Download artifacts for passing suites only.
+* `fail`: Download artifacts for failed suites only. (default value)
+
+---
+
+### `-b, --browserName`
+<p><small>| REQUIRED | STRING | WEB-APP ONLY |</small></p>
+
+The name of the browser in which to run tests.
+
+---
+
+### `--cypress.config`
+<p><small>| REQUIRED | STRING | CYPRESS ONLY |</small></p>
+
+The file path to the Cypress configuration file (typically `cypress.json`).
+
+---
+
+### `--device`
+<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
 
-__Description__: Executes tests based on information in the configuration file ([`.sauce/config.yml`](/testrunner-toolkit/configuration)).
+Find a real device for this test matching the device name or portion of the name provided. Use the [Get Devices API Request](https://docs.saucelabs.com/dev/api/rdc#get-devices) for a list of supported real devices in your region.
 
```

Comment:
In that specific case, we should provide an example, it's not as simple as specifying the device name.
For consistency, it's similar to the cli-driven approach for espresso.

There is different flags: 
```
id
name
platformName
platformVersion
carrierConnectivity
deviceType
private
```

Examples:
- Just filtering by name:
  ```
  --device "name=HTC.*"
  ```
- Several flags can be specified while being comma separated
   ```
  --device "name=HTC.*,platformVersion=8.0.0"
   ```

Reply:
OH!! That does change things. LOL I will make the clarification. Thanks for this.

Reply:
@FriggaHel -- are you sure the `platformName` is one of the supported flags for device? It is not one of the devices properties in the config file.